app/controllers/application_controller.rb

```diff
@@ -67,7 +67,10 @@ def user_id
   end
 
   def fake_ldap_attributes
-    Settings.fake_ldap_attributes.fetch(user_id, {}) if user_id && use_fake_ldap_attributes?
```

Comment:
Funny enough, I even used the phrase stringify keys in my commit message. :dizzy_face: